scaling_performance/optimizing_storage.adoc

```diff
@@ -59,7 +217,7 @@ VG=docker_vg
 ----
 +
 After the restart, `docker-storage-setup` sets up a volume group named
-`docker_vg` and creates a thin pool logical volume. Documentation for thin
+`docker_vg` and creates a thin-pool logical volume. Documentation for thin
```

Comment:
There was no hyphen in thin pool above - so either add up there, or remove here.

Reply:
Thin pool is a noun, and thin-pool is an adjective, correct?

Reply:
You are right, ignore my comment ;) hehe